build_sdk.py

```diff
@@ -30,33 +32,53 @@
 
 MICROKIT_EPOCH = 1616367257
 
-# Due to only supporting AArch64 (for now), we hard-code
-# the prefix of the toolchain used to compile seL4.
-TOOLCHAIN_PREFIX = "aarch64-none-elf-"
+TOOLCHAIN_AARCH64 = "aarch64-none-elf-"
```

Comment:
Maybe inline the strings into `c_toolchain()` and drop these constants completely?

Reply:
I like having them declared at the top so it's easily visible.